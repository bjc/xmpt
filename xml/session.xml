<iq type='set' id='session'>
  <session xmlns='urn:ietf:params:xml:ns:xmpp-session'/>
</iq>
