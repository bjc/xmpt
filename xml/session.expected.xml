<iq from='{bare_jid}' to='{full_jid}' id='session' type='result'/>
